src/services/tag-service.js

```diff
@@ -12,6 +12,13 @@ export class TagListService {
     });
   }
 
+  static del(index) {
```

Comment:
`static del(tag) {`
`return this.get().then(tags => {`
`   const updatedTags = tags.filter((elem) => elem!== tag);`
` this.set(updatedTags);`
` });`
`}`